index.html

```diff
@@ -13,7 +13,7 @@
     <header>
         <div class="navbar">
             <div class="nav-items flex-flow-row">
-                <div class="nav-item"><a><img src="#">Thrift store</a></div>
+                <div class="nav-item"><a><img src="images/logo.png" height="40rem" width="100rem"></a></div>
```

Comment:
Avoid inline styling here.

Reply:
I dont think inline styling takes rem as unit, I am not sure about it though. I tried adding height: 40px and width: 100px through console, so probably you can avoid inline style here.